src/lib/api.ts

```diff
@@ -16,7 +16,7 @@ const MAX_PAGES = 10
 const MAX_PER_PAGE = 100
 
 // @ts-expect-error TS(7023) FIXME: 'listSites' implicitly has return type 'any' becau... Remove this comment to see the full error message
-export const listSites = async ({ api, options }): SiteInfo[] => {
+export const listSites = async ({ api, options }): Promise<SiteInfo[]> => {
```

Comment:
I have the rest of this fixed in another draft PR. I just needed this to unblock errors elsewhere.